target-modules/pom.xml

```diff
@@ -25,15 +25,40 @@
         <mule.jms.connector.version>1.8.2</mule.jms.connector.version>
```

Comment:
Could you check if any other versions need update?

Reply:
I updated all the versions in the new revision. 

It also includes other changes because Windows builds were broken due to file streams/readers/writers not closing files properly and then other parts of the migration would fail trying to change the same files that were already open.